configuration/white_labels/base.py

```diff
@@ -0,0 +1,435 @@
+from screener.models import WhiteLabel
+
+
+class ConfigurationData:
+    is_default = False
+
+    @classmethod
+    def get_white_label(self) -> WhiteLabel:
+        raise NotImplemented()
+
+    public_charge_rule = {"link": ""}
+
+    more_help_options = {
+        "moreHelpOptions": [
+            {
+                "name": {"_default_message": "", "_label": ""},
```

Comment:
Why do we want these white label base values to be empty strings?

Reply:
The base is just to give an idea of what the structure should look like. It shouldn't be imported as part of the config.